src/Sentry.Unity/GameEventListener.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using UnityEngine;
+
+namespace Sentry.Unity
+{
+    public class GameEventListener : MonoBehaviour
+    {
+        public event Action? ApplicationEnter;
```

Comment:
Useful comment could be:

<summary>
Fires when a MonoBehavior OnApplicationPause is invoked 
</summary>
<remarks>
This hook doesn't fire on Android. Because ..
</remarks>